R/dm.R

```diff
@@ -147,10 +147,25 @@ validate_dm <- function(x) {
 #'
 #' @export
 cdm_get_src <- function(x) {
+  if (!is_dm(x)) abort_is_not_dm(class(x))
   tables <- cdm_get_tables(x)
   tbl_src(tables[1][[1]])
 }
 
+#' Get connection
+#'
+#' `cdm_get_con()` returns the connection object (`con`-part of \pkg{dplyr} source component) of a `dm`
+#' object.
+#'
+#' @rdname dm
+#'
+#' @export
+cdm_get_con <- function(x) {
+  con <- cdm_get_src(x)[["con"]]
+  if (is_null(con)) abort_no_con_if_local()
```

Comment:
Can we extract a `src` variable and use `inherits(src, "src_dbi")` instead?

Reply:
`class(src)` is `"src_local" "src"`; implemented